pkg/reconciler/revision/resources/pa.go

```diff
@@ -49,6 +49,19 @@ func MakePA(rev *v1alpha1.Revision) *av1alpha1.PodAutoscaler {
 				Name:       names.Deployment(rev),
 			},
 			ProtocolType: rev.GetProtocol(),
+			Reachability: func() av1alpha1.ReachabilityType {
+				// We don't know the reachability if the revision has just been created or is activating
+				cond := rev.Status.GetCondition(v1alpha1.RevisionConditionActive)
+				if cond != nil && cond.Status == corev1.ConditionUnknown {
+					return av1alpha1.ReachabilityUnknown
+				}
+
+				if rev.IsReachable() {
+					return av1alpha1.ReachabilityReachable
+				} else {
```

Comment:
remove else an unindent.

Reply:
Nit?

Reply:
Readability/style (also an error if you run staticcheck over it).

Reply:
Sounds like a nit. 

I left it this way intentionally for readability, let me know if you feel strongly about this.

Reply:
If you don't really want to change it, I'll fix it in the next few minutes after this lands. Up to you :-)

Reply:
> also an error if you run staticcheck over it

Which `staticcheck` [error](https://staticcheck.io/docs/checks) are you seeing? It's not picking up anything on my machine.

Reply:
Sorry it's linter error (along these lines: https://github.com/golang/lint/blob/master//lint.go#L1096). There's implied return.